pages/StartView.page.ts

```diff
@@ -10,7 +10,7 @@ export class StartPage{
     constructor(page){
         this.page = page;  
         this.text_h2_Ueberschrift = page.getByTestId('all-service-provider-title');
-        this.card_item_email = page.locator('[href="https://de.wikipedia.org/wiki/E-Mail"]');
+        this.card_item_email = page.locator('[href="https://de.wikipedia.org/wiki/E-Mail"]').first();
```

Comment:
In diesem Scope können wir nichts ändern, aber langfristig finde ich das ziemlich unsinnig. Kannst du evtl. mal mit Christian oder Kristoff sprechen, dass die Seed-Daten vernünftige Inhalte bekommen, statt nur copy-paste das gleiche zu sein?

Reply:
Habe ich gestern mit Christian und Cornelius besprochen. Die URLs sind demnächst alle eindeutig.